ambry-api/src/main/java/com/github/ambry/account/AccountService.java

```diff
@@ -130,6 +130,18 @@ default Container getContainer(String accountName, String containerName) throws
     return account != null ? account.getContainerByName(containerName) : null;
   }
 
+  /**
+   * Get an existing container from a given account.
+   * @param accountName the name of account which container belongs to.
+   * @param containerId the id of container to get.
+   * @return the requested {@link Container} object or null if not present.
+   * @throws AccountServiceException if exception occurs when getting container.
+   */
+  default Container getContainer(String accountName, Short containerId) throws AccountServiceException {
```

Comment:
How about calling this `getContainerById`? And change line 128 to `getContainerByName`?